### PR TITLE
Fix RSSLink integration. Fixes #56

### LINKED ISSUE #56: RSSlink breaks rendering
```
$ hugo server -w --theme=sabayon-website-next-theme
WARNING: Site config's rssURI is deprecated and will be removed in a future release. Set baseName in outputFormats.RSS.
Started building sites ...               
ERROR 2017/09/21 22:11:21 Error while rendering "page": template: theme/_default/single.html:4:11: executing "theme/_default/single.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "page": template: theme/release/single.html:4:11: executing "theme/release/single.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "page": template: theme/article/single.html:4:11: executing "theme/article/single.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "taxonomy": template: theme/indexes/tag.html:4:11: executing "theme/indexes/tag.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "taxonomy": template: theme/indexes/category.html:4:11: executing "theme/indexes/category.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "home": template: index.html:4:11: executing "index.html" at <partial "meta.html" ...>: error calling partial: template: partials/meta.html:9:6: executing "partials/meta.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
ERROR 2017/09/21 22:11:21 Error while rendering "404": template: theme/404.html:12:14: executing "theme/404.html" at <.RSSlink>: can't evaluate field RSSlink in type *hugolib.PageOutput
```

### PR DESCRIPTION
**Pull request by Ryuno-Ki**

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -411,7 +411,6 @@ <h2 class="text-center" style="margin-bottom:21.4333px;">Our sponsors:</h2>
             </div><!-- .container -->
         </section><!-- #getit.services -->
 
-    {{ partial "footer.copyright.html" . }}
```
Comment (Ryuno-Ki) [inline layouts/index.html:414]: That file was not found any more oO

File: config.toml [hunk 2]
```diff
@@ -1,5 +1,4 @@
-baseurl = "https://next.sabayon.org/"
+baseurl = "https://www.sabayon.org/"
```
Comment (mudler) [inline config.toml:1]: Don't!! :)

**Comment by mudler** [comment]
Maybe this is not needed anymore since: https://github.com/Sabayon/sabayon-website-next-theme/commit/cf2d61e611dcc87af3f668dca252462db30b8be1

**Comment by Ryuno-Ki** [comment]
OMG! This PR is still open? I'm closing it as Won't Fix!

**Pull request closed without merging**